app/views/admin/loan_questions/_loan_types.slim

```diff
@@ -0,0 +1,35 @@
+.required-loan-types
+  label = t('loan_questions.required_loan_types')
+  .actions
+    a[data-expands='loan-types']
+      i.fa.fa-eye.fa-large
+      span = t('common.show')
+    a[data-hides='loan-types']
+      i.fa.fa-eye-slash
+      span = t('common.hide')
+
+  div data-expandable='loan-types'
+    - if @loan_question.parent
+      = f.collection_radio_buttons :override_associations,
+        [[false, t('loan_questions.inherit_parent')],
+        [true, t('loan_questions.override_parent')]], :first, :last,
+        item_wrapper_class: 'radio'
+
+    - hideLoanTypes = @loan_question.parent && !@loan_question.override_associations ? 'hidden': ''
+    p.loan-types-description = t('loan_questions.loan_types.description')
+    p.loan-types-description = t('loan_questions.loan_amounts.description')
+    .loan-types-container class="#{hideLoanTypes}"
+      .form-horizontal
+        - cfreqs = f.object.custom_field_requirements.sort_by { |i| i.loan_type.label.text }
```

Comment:
This should not be happening in the view. Also there is a Law of Demeter violation here, but that should be the concern of the backend developer.